draft-ietf-tls-tls13.md

```diff
@@ -1395,7 +1395,7 @@ Structure of this message:
        uint8 CipherSuite[2];    /* Cryptographic suite selector */
 
        struct {
-           ProtocolVersion max_supported_version = { 3, 4 };    /* TLS v1.3 */
+           ProtocolVersion legacy_version = { 3, 3 };    /* TLS v1.2 */
```

Comment:
And while we’re out it, yes, let’s just make it an integer so it maps to whatever x.y or x or /x we use in the future.